lib/metrics/cyclomatic_complexity_metric.dart

```diff
@@ -2,19 +2,27 @@ library cyclomatic_complexity_metric;
 
 import 'package:analyzer/error/listener.dart';
 import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:solid_metrics/models/cyclomatic_complexity_parameters.dart';
 import 'package:solid_metrics/visitor/cyclomatic_complexity_visitor.dart';
 
 /// A Complexity metric checks content of block and detects more easier solution
 class CyclomaticComplexityMetric extends DartLintRule {
-  static const _lintCode = LintCode(
+  /// lint code of metric
+  static const lintCode = LintCode(
     name: 'cyclomatic_complexity_metric',
     problemMessage: 'Check complexity',
   );
 
+  /// Additional parameteres
```

Comment:
```suggestion
  /// The additional parameters for this metric.
```
[Effective Dart](https://dart.dev/effective-dart/documentation#prefer-starting-a-non-boolean-variable-or-property-comment-with-a-noun-phrase)